src/app/modules/main/wallet_section/collectibles/module.nim

```diff
@@ -81,49 +81,36 @@ method currentCollectibleModuleDidLoad*(self: Module) =
   self.checkIfModuleDidLoad()
 
 method fetchOwnedCollectibles*(self: Module) =
-  self.controller.fetchOwnedCollectibles(self.chainId, self.address)
+  self.controller.fetchOwnedCollectibles(self.chainId, self.addresses)
```

Comment:
so, fetch is now triggered at the same time for all addresses. They will be appended to the list in whichever random order the async tasks finish. Do we want to apply some sorting to keep things consistent? Though that means appended collectibles will now possibly be inserted in the middle of the list

Reply:
I think this is why we need to apply same logic as for transaction (scheduler + db + query) Doing all that client side is a bit nonsense I believe

Reply:
allright, stuff might look a bit funky until we do the big rework, just to know we're OK with that.